modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -270,8 +272,10 @@ public void getM2MAccessToken() throws Exception {
     @Test(dependsOnMethods = "getM2MAccessToken")
     public void testSelfOnboardOrganization() throws IOException {
 
+        organizationHandle = System.currentTimeMillis() + "-" + ORG_HANDLE_GREATER_HOSPITAL;
```

Comment:
Now, don't we have a test to verify that org creation API works without orgHandle?